code/__DEFINES/misc.dm

```diff
@@ -255,10 +255,11 @@
                             0, 0, 1, 0,\
                             0, 0, 0, 1)
 
-#define MATRIX_GREYSCALE list(0.3, 0.3, 0.3, 0,\
-                              0.3, 0.3, 0.3, 0,\
-                              0.3, 0.3, 0.3, 0,\
-                              0,   0,   0,   1)
+#define MATRIX_GREYSCALE list(0.33, 0.33, 0.33, 0,\
```

Comment:
Last I checked, matrices were a 16-entry 4x4? What does making the matrix a 20-entry 4x5 accomplish?